bundle/Resources/translations/messages.de.yml

```diff
@@ -0,0 +1,3 @@
+field_definition:
```

Comment:
Can you use a single line for every translation, like `field_definition.enhancedbinaryfile.allowedTypes` ?